slycot/tests/test.py

```diff
@@ -64,6 +64,111 @@ def test_td04ad_static(self):
             nr,a,b,c,d = transform.td04ad(rc,nin,nout,index,den,num)
 
 
+# ===================================================
+# Begin tb05ad tests
+import numpy as np
+
+from numpy.testing import assert_raises, assert_almost_equal
+from numpy.testing import assert_array_almost_equal, dec
+
+CASES = {}
+
+CASES['fail1'] = {'A': np.array([[-0.5,  0.,  0.,  0. ],
+                              [ 0., -1.,  0. ,  0. ],
+                              [ 1.,  0., -0.5,  0. ],
+                              [ 0.,  1.,  0., -1. ]]),
+                   'B': np.array([[ 1.,  0.],
+                             [ 0.,  1.],
+                             [ 0.,  0.],
+                             [ 0.,  0.]]),
+                   'C': np.array([[ 0.,  1.,  1.,  0.],
+                             [ 0.,  1.,  0.,  1.],
+                             [ 0.,  1.,  1.,  1.]])}
+
+n = 20
+p = 10
+m = 14
+np.random.seed(40)
+
+CASES['pass1'] = {'A': np.random.rand(n, n),
+                  'B':  np.random.rand(n, m),
+                  'C':  np.random.rand(p, n)}
+
+
+class test_tb05ad(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_tb05ad_ng(self):
+        for key in CASES:
```

Comment:
I wasn't aware of this recommendation, and I've never really thought about the usefulness of test method docstrings, so I suppose I was expressing a personal preference---sorry.

I don't think there's consensus on whether or not to have such docstrings (see, e.g., [this Stackoverflow question](http://stackoverflow.com/q/1726622/1008142)).  python-control has them in some tests, and not in others, so unfortunately we can't get guidance there.

There's a [nose plug-in](https://github.com/MarechJ/nosenodocstrings) that can prevent the behaviour objected to in that numpy reference.